log/field.go

```diff
@@ -152,6 +153,26 @@ func Lazy(ll LazyLogger) Field {
 	}
 }
 
+// Skip creates a no-op log field that should be ignored by the tracer.
+// It can be used to capture optional fields, for example those that should
+// only be logged in non-production environment:
+//
+//     func customerField(order *Order) log.Field {
+//          if os.Getenv("ENVIRONMENT") == "dev" {
+//              return log.String("customer", order.Customer.ID)
+//          }
+//          return log.Skip()
+//     }
+//
+//     span.LogFields(log.String("event", "purchase"), customerField(order))
+//
+func Skip() Field {
+	return Field{
+		key:       "_",
```

Comment:
this can be skipped, zero string value is just as good, we only need to initialize the fieldType